flang/test/Lower/intrinsic-procedures.f90

```diff
@@ -323,6 +340,60 @@ subroutine iand_test(a, b)
   ! CHECK: %{{[0-9]+}} = and %{{[0-9]+}}, %{{[0-9]+}} : i{{(8|16|32|64|128)}}
 end subroutine iand_test
 
+! IBCLR
+! CHECK-LABEL: ibclr_test
+function ibclr_test(i, j)
+  ! CHECK: %[[VAL_0:.*]] = fir.alloca i32 {bindc_name = "ibclr_test"
+  ! CHECK: %[[VAL_1:.*]] = fir.load %[[VAL_2:.*]] : !fir.ref<i32>
+  ! CHECK: %[[VAL_3:.*]] = fir.load %[[VAL_4:.*]] : !fir.ref<i32>
```

Comment:
Auto generated tests are quite weak regarding function argument capture. The load order is quite fragile, and for a human reader, there is no hope matching `i`, `j` to the values and assessing the produced code.
I would still manually capture the arguments here instead and replace the `fir.load %[[VAL_X:.*]]` capture with it.

I also replace the entry fir.load / constant  `! CHECK:` with `! CHECK-DAG` because the order of those is unrelated to (and I am not sure what is controlling it exactly).

Reply:
Yeah, auto generation is almost as problematic as hand generation.  For this group of tests I've switched to `argN` and `result`, and switched most `CHECK`s to `CHECK-DAG`s, rather than just the ones in the **"prolog".__**

Reply:
I would not have replaced all the CHECK by CHECK-DAGs (for the ones the order of instructions is controlled by the code being tested, I think it is fine to have CHECKs. I am not sure what are the implication of having everything as CHECK-DAGs). But that's probably OK, not need to revert that here.